src/components/VDynamicForm.vue

```diff
@@ -1,82 +1,123 @@
+        <ul>
+            <li>Resolve Tooltip! -> Probably just pull in a package to handle this!</li>
+            <li>Resolve the ability to put the Slot on FormLabel in the Label element or to put it underneath (For
+                Radio/Checkbox Buttons)
+            </li>
+            <li>Build up some example forms that are the same as previously discussed</li>
+        </ul>
 
-    <hr>
-  <div class="container items-center px-5 py-12 lg:px-20">
-    <div class="flex flex-col w-full p-10 px-8 pt-6 mx-auto my-6 mb-4 transition duration-500 ease-in-out transform bg-white border rounded-lg lg:w-1/2">
-      <div class="lg:text-center">
-        <p class="mt-2 text-3xl leading-8 font-extrabold tracking-tight text-gray-900 sm:text-4xl" v-if="schema.title" v-text="schema.title"></p>
-        <p class="mt-4 max-w-2xl text-xl text-gray-500 lg:mx-auto" v-if="schema.subtitle" v-text="schema.subtitle"></p>
-        <p v-if="schema.description" v-text="schema.description"></p>
-      </div>
-      <form ref="form">
-        <div v-for="formInput in schema.fields" :key="formInput.id">
-          <keep-alive>
-            <component
-                :is=formInput.type
-                v-bind="formInput"
-                @updated="updateFormData"
-                @submit="triggerSubmit"
-            >
-            </component>
-          </keep-alive>
+        <hr>
+        <div class="container items-center px-5 py-12 lg:px-20">
+            <div
+              class="flex flex-col w-full p-10 px-8 pt-6 mx-auto my-6 mb-4 transition duration-500 ease-in-out transform bg-white border rounded-lg lg:w-1/2">
+                <div class="lg:text-center">
+                    <p class="mt-2 text-3xl leading-8 font-extrabold tracking-tight text-gray-900 sm:text-4xl"
+                       v-if="form.title" v-text="form.title"></p>
+                    <p class="mt-4 max-w-2xl text-xl text-gray-500 lg:mx-auto" v-if="form.subtitle"
+                       v-text="form.subtitle"></p>
+                    <p v-if="form.description" v-text="form.description"></p>
+                </div>
+                <form ref="form">
+                    <div v-for="group in form.groups">
+                        <div v-for="formInput in group.fields" :key="formInput.id">
+                            <keep-alive>
+                                <component
+                                  :is="getComponentFromType(formInput.type)"
+                                  v-bind="formInput"
+                                  @updated="updateFormData"
+                                  @submit="triggerSubmit"
+                                >
+                                </component>
+                            </keep-alive>
+                        </div>
+                    </div>
+                </form>
+            </div>
         </div>
-      </form>
     </div>
-  </div>
-  </div>
 </template>
 
 <script>
+import VRadio from './atomic/dynamic-form/VRadios';
```

Comment:
As all of the components are globally loaded then you don't need to load them all here :)

Reply:
I think we still need to - for the portal, they're all loaded as e.g. <p-radio>, but we refer to them as `VRadio` within the UI kit. Could be worth a try, but I think registering them will mean we can load components with compatible prefixes without messing up the UI